lib/brain.js

```diff
@@ -298,9 +304,11 @@ Brain.prototype.checkWifiStatus = function checkWifiStatus () {
   this._transitionState('booting')
 
   this.wifi.status(function (err, status, ip) {
-    if (err || status === 'bill') {
-      if (err) console.log(err.stack)
-      if (self.state !== 'wifiConnecting') self._wifiConnecting()
+    if (err) {
```

Comment:
What about handling ``status === 'bill'``? If this is not actually necessary, explain why.

Reply:
status === 'bill' was actually redundant since quite some time, the linked commit mistakenly replaced 'pending' with 'bill' in this line

https://github.com/lamassu/lamassu-machine/commit/54b0085f351182f53174751a533d96f3a3640b6c#diff-aa306c9a688d1483cc340067b667e2ffR214